src/main/java/lotto/Domain/LottoResult/Prize.java

```diff
@@ -0,0 +1,36 @@
+package lotto.Domain.LottoResult;
+
+public enum Prize {
+    FIRST_PRIZE(0, 2000000000, 6),
```

Comment:
이렇게 숫자 자리수가 많아질 때는 `_`로 자리수를 구분해 줄 수 있습니다!
```suggestion
    FIRST_PRIZE(0, 2_000_000_000, 6),
```

Reply:
좋은 말씀들 감사합니다!

`_`로 구분하는건 이번에 처음 알았네요 ㅠㅠ 다음미션에는 이용해보도록 하겠습니다!
또한 저도 외부 API 에 의존하였기에 테스트하기 곤란하다고 느꼈는데 배워갑니다!